internal/wrapper/wrapper.go

```diff
@@ -53,6 +53,12 @@ func RunTerraform(args []string, arch string) (int, error) {
 
 	log.Printf("version '%s' matches all constraints", matchingRelease.Version)
 
+	err = checkStateCommand(args, matchingRelease.Version)
+	if err != nil {
+		log.SetOutput(os.Stderr)
+		log.Fatal("error: ", err)
```

Comment:
I don't think you need to do this, since the thing that calls `RunTerraform` will automatically print any errors: https://github.com/etsy/terraform-demux/blob/7a90e7e2eedc83936a5c82edf2e39c38d6a2111f/cmd/terraform-demux/main.go#L29-L35